frontend/app/src/components/accounts/BlockchainBalances.vue

```diff
@@ -33,14 +33,17 @@
         :primary-action="tc('common.actions.save')"
         :secondary-action="tc('common.actions.cancel')"
         :action-disabled="!valid"
+        :loading="isAccountOperationRunning().value || pending"
```

Comment:
Not for this PR but I think it would make sense for this places we have functions that return computed properties we should start extracting sub-components